gtsfm/data_association/data_assoc.py

```diff
@@ -121,7 +128,15 @@ def run(
             else:
                 per_rejected_track_avg_errors.append(avg_track_reproj_error)
 
-        track_cheirality_failure_ratio = num_tracks_w_cheirality_exceptions / len(tracks_2d)
+        # aggregate the exit codes
+        exit_codes_wrt_computed_distribution = Counter(exit_codes_wrt_computed)
```

Comment:
Hi @ayushbaid, thanks for the PR. this ew lines from 131-135 are a bit confusing to me. Could you add some inline comments explaining what's going on in a bit more detail? why do we need the `var1_and_var2` variables?